packages/assets/scss/01-atoms/_divider.scss

```
@@ -50,3 +44,11 @@
     }
   }
 }
+
+.ma__org-page .ma__divider,
+.ma__organization-detail .ma__divider{
+  @include ma-container;
+
+//  padding: 0;
```

```suggestion
//  padding: 0;
```
looks good, can you just clean this line up